README.md

```diff
@@ -16,6 +16,12 @@
 
 ### Tests lokal ausführen:
 
+#### Lokale Installation
```

Comment:
Ab Zeile 76 ist das schon dokumentiert(## Playwright installieren unter Windows)

Reply:
Ich hab das bestehende mal nach oben gezogen, weil die Installation in einer Readme meist oben steht